include/mplib/planning_world.h

```diff
@@ -166,6 +167,16 @@ class PlanningWorldTpl {
     normal_object_map_[name] = collision_object;
   }
 
+  /**
+   * Adds a normal object (``CollisionObjectPtr``) with given name to world
+   *
+   * @param name: name of the collision object
+   * @param collision_object: collision object to be added
+   */
+  // TODO(merge): remove name
```

Comment:
What is this todo for?

Reply:
FCLObject already contains name. No need to require an extra argument to `addNormalObject`.
For the overloaded `addNormalObject` taking a `CollisionObjectPtr` as input, the name is still required.